lib/nat-traversal.js

```diff
@@ -0,0 +1,167 @@
+const isBrowser = typeof process === 'undefined' || process.type === 'renderer' || process.browser === true || process.__nwjs
+
+if (isBrowser) {
+  module.exports = null
+} else {
+  const arrayRemove = require('unordered-array-remove')
+  const debug = require('debug')('webtorrent:nat-traversal')
+  const natUpnp = require('nat-upnp') // browser exclude
+  const natpmp = require('nat-pmp') // browser exclude
+  const network = require('network') // browser exclude
```

Comment:
this should really not be here, move them to the top

Reply:
why? they are conditionally required